.circleci/config.yml

```diff
@@ -12,22 +12,44 @@ parameters:
     type: string
     default: dps-releases
 
+executors:
+  docker-env:
+    docker:
+      - image: cimg/openjdk:21.0
+      - image: localstack/localstack:3
+        environment:
+          - SERVICES=s3,sqs,sns
+          - ES_PORT_EXTERNAL=4571
+          - DEBUG=${DEBUG- }
+          - DATA_DIR=/tmp/localstack/data
+          - DOCKER_HOST=unix:///var/run/docker.sock
+          - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
+          - AWS_EXECUTION_ENV=True
+          - DEFAULT_REGION=eu-west-2
+          - TMPDIR=/private
+      - image: postgres:15
```

Comment:
This is an oversight on my part when I did the upgrade from 15-16. Should be using the postgres:16 image